softserve/lib.py

```diff
@@ -43,60 +40,59 @@ def create_node(counts, name, node_request, pubkey):
     '''
     Create a node in the cloud provider
     '''
-    driver = get_driver(Provider.RACKSPACE)
-    conn = driver(
-        app.config['USERNAME'],
-        app.config['API_KEY'],
-        region=app.config['AUTH_SYSTEM_REGION']
-    )
-    flavor = conn.ex_get_size('performance1-2')
-    image = conn.get_image('8bca010c-c027-4947-b9c9-adaae6e4f020')
-
-    # Terrible hack to workaround libcloud bug #1011
-    # On python 3 a unicode string should be str. On python2, we will have to
-    # force unicode to str. Otherwise libcloud doesn't recognize it.
-    if not isinstance(pubkey, str):
-        pubkey = str(pubkey)
-
-    step = SSHKeyDeployment(pubkey)
-    node_request = NodeRequest.query.get(node_request)
+    conn = boto.ec2.connect_to_region(app.config['REGION_NAME'])
+    conn.import_key_pair(name, pubkey)
     for count in range(int(counts)):
         vm_name = ''.join(['softserve-', name, '.', str(count+1)])
-        node = conn.deploy_node(
-            name=vm_name, image=image, size=flavor, deploy=step
-        )
-        for ip_addr in node.public_ips:
-            try:
-                socket.inet_pton(socket.AF_INET, ip_addr)
-                network = ip_addr
-            except socket.error:
-                continue
-        machine = Vm(ip_address=network,
+
+        reservation = conn.run_instances(
+                      'ami-04f22a6831d585e63',
+                      key_name=name,
+                      instance_type='t3.small',
+                      security_groups=['regression'])
+
+        instance = reservation.instances[0]
+
+        # wait for instance to be running
+        while instance.update() != "running":
+            time.sleep(5)
+
+        # add instance tag
+        instance.add_tag("Name", vm_name)
```

Comment:
As we may in the future share the account with others, maybe it would be useful to also add a tag so we can see what are our servers.